metabase/metabase
# big query schemas settings

Update styles of BigQuery schemas settings.

Before:
<img width="658" alt="Screenshot 2022-01-14 at 17 23 44" src="https://user-images.githubusercontent.com/14301985/149558253-bf37574e-2473-403b-93b2-f504c73c62c1.png">

After:
<img width="643" alt="Screenshot 2022-01-14 at 17 24 01" src="https://user-images.githubusercontent.com/14301985/149558204-f719e46c-bd9c-4414-90ca-311f119ad43e.png">

### How to verify

- Try add BigQuery database (using the new driver)
- Ensure "Datasets" fields allows to add all schemas or specify included/excluded pattern
